### ISSUE TITLE
validate on test set?

### ISSUE THREAD

**Issue by chronbird** [raoyongming/PointGLR#3]
Hi,

I find your code validating test set of Modelnet during training for the unsupervised classification task, can you help me understand how you choose the best model? Thanks.

**Comment by raoyongming** [comment]
Hi, 

Thanks for your interest in our work.

The implementation follows RelationShapeCNN (details can be found [here](https://github.com/Yochengliu/Relation-Shape-CNN/blob/master/train_cls.py)).  I also see the same implementation in [PointNe++](https://github.com/charlesq34/pointnet/blob/master/train.py) and [PointNet](https://github.com/charlesq34/pointnet/blob/master/train.py). Since there is no official validation set in ModelNet, I think it is a common practice in this area. 

Although I agree that using an extra validation set is better, I think the comparison with the previous methods in our experiments is fair.
